File: Interviews/LeetCode/Daily-Problems/2022/November/Day-08/1544.cpp
class Solution {
   public:
    string makeGood(string s) {
        string ans = "";
        for (char c : s) {
            if (ans.size() == 0 or abs(int(c) - int(ans.back())) != 32) {
                ans.push_back(c);
            } else {
                ans.pop_back();
            }
        }

        return ans;
    }
};

File: Interviews/LeetCode/Daily-Problems/2022/November/Day-08/1544.py
class Solution:
    def makeGood(self, s: str) -> str:
        ans = []
        for c in s:
            if len(ans) == 0 or abs(ord(c) - ord(ans[-1])) != 32:
                ans.append(c)
            else:
                ans.pop()

        return "".join(ans)
